docs/api/web-contents.md

```diff
@@ -1624,24 +1625,26 @@ The `landscape` will be ignored if `@page` CSS at-rule is used in the web page.
 An example of `webContents.printToPDF`:
 
 ```js
-const { BrowserWindow } = require('electron')
+const { app, BrowserWindow } = require('electron')
 const fs = require('node:fs')
 const path = require('node:path')
 const os = require('node:os')
 
-const win = new BrowserWindow()
-win.loadURL('https://github.com')
+app.on('ready', () => {
```

Comment:
Did a quick search in this repo and it looks like most examples use `app.whenReady()` instead of listening for the `ready` event directly, so maybe we should update this for consistency.

Reply:
@erikian good call! done in 95cf590357